python/utilities/log_list/print_log_list.py

```diff
@@ -39,9 +40,9 @@
 
 def is_log_list_valid(json_log_list, schema_file):
     try:
+        schema_data = json.load(open(schema_file, "rb"))
         jsonschema.validate(
-            json_log_list,
-            json.load(open(schema_file, "rb")))
+            json_log_list, schema_data)
```

Comment:
I kind of like the old version better here, especially considering Python's mad function scoping... Not a big deal, if you prefer this, can keep it like this.